packages/openapi/api/definitions/playlists.yaml

```
@@ -0,0 +1,574 @@
+        412:
+          description: There is already an active Playlist for the studio that the Playlist belongs to.
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  status:
+                    type: number
+                    example: 412
+                  message:
+                    type: string
+                    example: Rundown Playlist is active, please deactivate before preparing it for broadcast
+        500:
+          $ref: '#/components/responses/internalServerError'
+  deactivate:
+    put:
+      operationId: deactivate
+      tags:
+        - playlists
+      summary: Deactivates a Playlist.
+      parameters:
+        - name: playlistId
+          in: path
+          description: Playlist to deactivate.
+          required: true
+          schema:
+            type: string
+      responses:
+        200:
+          $ref: '#/components/responses/putSuccess'
+        404:
+          $ref: '#/components/responses/playlistNotFound'
+        500:
+          $ref: '#/components/responses/internalServerError'
+  executeAdLib:
+    put:
+      operationId: executeAdLib
+      tags:
+        - playlists
+      summary: Executes the requested AdLib/AdLib Action. This is a "planned" AdLib (Action) that has been produced by the blueprints during the ingest process.
+      parameters:
+        - name: playlistId
+          in: path
+          description: Playlist to execute adLib in.
+          required: true
+          schema:
+            type: string
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                actionType:
+                  type: string
+                  description: An actionType string to specify a particular variation for the AdLibAction, valid strings are to be read from the status API
+                adLibId:
+                  type: string
+                  description: AdLib to execute
+              required:
+                - adLibId
+              example:
+                actionType: pvw
+      responses:
+        200:
+          $ref: '#/components/responses/executeActionSuccess'
+        404:
+          $ref: '#/components/responses/playlistNotFound'
+        412:
+          description: Specified Playlist is not active, there is not an on-air Part instance or an adLib for the provided `adLibId` cannot be found.
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  status:
+                    type: number
+                    example: 412
+                  message:
+                    type: string
+                    example: AdLib could not be found!
+        500:
+          $ref: '#/components/responses/internalServerError'
+  moveNextPart:
+    put:
```

Like above, maybe should be `post`?
```suggestion
    post:
```